### PR TITLE
fetch-node-v5-upgrade

### PR DESCRIPTION
**Pull request by dhruvTor**
fetch-node-details-sdk has been upgraded to V5 
Changes suggested by @metallicalfa2 have been done also caching is now fixed

### CODE REVIEW

File: Sources/FetchNodeDetails/SampleOutput.swift [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+//  Created by Dhruv Jaiswal on 04/04/22.
+//
+
+import Foundation
+import BigInt
+import web3
+@testable import web3
+
+
+struct SampleOutputMainet{
```
Comment (metallicalfa2) [inline Sources/FetchNodeDetails/SampleOutput.swift:14]: fix spelling mistakes

File: Sources/FetchNodeDetails/SampleOutput.swift [hunk 2]
```diff
@@ -0,0 +1,152 @@
+//
```
Comment (metallicalfa2) [inline Sources/FetchNodeDetails/SampleOutput.swift:1]: Don't bundle the sample files in main package. Keep them in correct folders

File: Sources/FetchNodeDetails/SampleOutput.swift (hunk 1 above)
Comment (dhruvTor) [reply Sources/FetchNodeDetails/SampleOutput.swift]: Done

File: Sources/FetchNodeDetails/SampleOutput.swift (hunk 2 above)
Comment (dhruvTor) [reply Sources/FetchNodeDetails/SampleOutput.swift]: Done

**Review by metallicalfa2 — APPROVED** [review]
LGTM.

**Pull request merged**